src/main/data_structure/list_exercise.py

```diff
@@ -14,15 +14,14 @@ def find_average(list_of_number):
     return summation / len(list_of_number)
 
 
-def sort_list(list_of_number):
-    
+def list_sort(list_of_number):
     sorted_list = sorted(list_of_number)
     logger.info("Sorted list : {}".format(sorted_list))
-    
+
     return sorted_list
 
 
-def slicing(list1):
+def list_slicing(list1):
```

Comment:
function name should be verb like slice_list
change variable name list1 to meaningful name